integration-tests/scripts/firehose-connector.sh

```diff
@@ -0,0 +1,13 @@
+CURRENT_DIR=$(pwd)
+FIREHOSE_CONNECTOR_PATH=$CURRENT_DIR/testnet/mx-chain-ws-connector-firehose-go/cmd/connector
+
+cd ../../devel/standard/
+DEVEL=$(pwd)
```

Comment:
what is DEVEL ?

Reply:
The directory name where the firehose expects to find the ws connector binary to start it within their ingestion node process